lib/modules/datasource/glasskube-packages/index.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { getPkgReleases } from '..';
+import { Fixtures } from '../../../../test/fixtures';
+import * as httpMock from '../../../../test/http-mock';
+import { GlasskubePackagesDatasource } from '.';
+
+describe('modules/datasource/glasskube-packages/index', () => {
+  const versionsUrl = new URL(
+    `${GlasskubePackagesDatasource.defaultRegistryUrl}/cloudnative-pg/versions.yaml`,
+  );
+  const versionsYaml = Fixtures.get('versions.yaml');
+  const packageManifestUrl = new URL(
+    `${GlasskubePackagesDatasource.defaultRegistryUrl}/cloudnative-pg/v1.23.1+1/package.yaml`,
+  );
+  const packageManifestYaml = Fixtures.get('package.yaml');
+  const packageManifestNoReferencesYaml = Fixtures.get(
+    'package_no_references.yaml',
+  );
+
+  it('should handle error response on versions request', async () => {
+    httpMock
+      .scope(versionsUrl.origin)
+      .get(versionsUrl.pathname)
+      .reply(500, 'internal server error');
+    await expect(
+      getPkgReleases({
+        datasource: GlasskubePackagesDatasource.id,
+        packageName: 'cloudnative-pg',
+        registryUrls: [GlasskubePackagesDatasource.defaultRegistryUrl],
```

Comment:
I think this should be removed, the datasource doesn't seem to support custom registry urls.

if it supports then, then use different urls for testing.

otherwise remove registry urls from tests.

Reply:
The datasource should be able to support custom registries. Therefore, the tests should use different URLs. Did I understand that correctly?

Reply:
Your interpretation is correct, yeah.